docs/layers/tile-3d-layer.md

```diff
@@ -100,11 +109,6 @@ This value is only applied when [tile format](https://github.com/AnalyticalGraph
 
 - A URL to fetch tiles entry point [Tileset JSON](https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/specification#tileset-json) file or json object of tileset.
 
-##### `_ionAssetId` (Number|String, Optional)
-##### `_ionAccessToken` (String, Optional)
-
-- `_ionAssetId` and `_ionAccessToken` are used to fetch ion dataset. They are experimental properties, may change in next releases. 
-
 [Set up Ion account](https://cesium.com/docs/tutorials/getting-started/#your-first-app);
```

Comment:
Seems a little weird to have this line here when you cut the `ion` headers. Maybe place it somewhere else?